Tests/FunctionalTests/MiscellaneousTests.swift

```diff
@@ -536,4 +536,17 @@ class MiscellaneousTestCase: XCTestCase {
             }
         }
     }
+
+    func testExecutableTargetMismatch() {
+        fixture(name: "Miscellaneous/TargetMismatch") { path in
+            do {
+                let output = try executeSwiftBuild(path)
+                print(output)
```

Comment:
What's our general stance on print it from unit test?￼. I have a couple in unit tests that I wrote that I would like to remove;  but I'm not sure whether it's better to leave them in or to remove them.

Ideally we'd have a verbose mode for unit tests.

Reply:
oops this is a leftover